components/html-block/html-block.js

```diff
@@ -0,0 +1,271 @@
+import '../colors/colors.js';
+import { css, LitElement } from 'lit-element/lit-element.js';
+
+let mathJaxLoaded;
+
+const loadMathJax = () => {
+
+	window.MathJax = {
+		options: {
+			menuOptions: {
+				settings: { zoom: 'None' }
+			}
+		},
+		chtml: { scale: 1 },
```

Comment:
I think the default configuration for `chtml: { scale }` is 1. Does this need to be set explicitly?

Reply:
I think I can remove this.  Thanks.  :)